### DISCUSSION TITLE
[工具学习]Latex工具学习

### DISCUSSION THREAD

**Discussion by bifenglin** [X-lab2017/open-wonderland#57] [2️⃣ 聊项目]
由华东师范大学数学学院的潘建瑜老师整理，包含一些论文试卷等模版可以直接使用，并且内容清晰全面，适合需要了解学习Latex的同学学习。
地址：[https://math.ecnu.edu.cn/~jypan/Latex/index.html](https://math.ecnu.edu.cn/~jypan/Latex/index.html)
截图：
![image](https://user-images.githubusercontent.com/15813364/170640209-1fc5ecd9-03ce-4368-8bb4-2921d3ce61af.png)

**Comment by will-ww** [comment]
挺不错的，收入知识库~
